staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -702,6 +702,7 @@ func Test_syncNode(t *testing.T) {
 							Effect: v1.TaintEffectNoSchedule,
 						},
 					},
+					ProviderID: "fake://12345",
```

Comment:
This test case name is "provider ID not implemented", so I think the existing node is not supposed to have a provider ID and the updated node is not supposed to have it either.